### PR TITLE
feedbox-fix container svg hover + BTN SEND fix

### PR DESCRIPTION
**Pull request by MYGelo**

### CODE REVIEW

**Review by Pohorielova — APPROVED** [review]

File: src/sass/components/_button.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.btn__item{
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:1]: ok

File: src/sass/components/_button.scss [hunk 2]
```diff
@@ -16,29 +16,30 @@
 
   background-color: $accent-color;
   border: none;
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:19]: ok

File: src/sass/components/_button.scss [hunk 3]
```diff
@@ -16,29 +16,30 @@
...
-          
+
   background-size: 200% 100%;
-  background-image: linear-gradient(to right, transparent 50%, #B5302D 50%);
-  transition: background-position .3s cubic-bezier(0.19, 1, 0.22, 1) .1s, color .5s ease 0s, background-color .5s ease;
+  background-image: linear-gradient(to right, transparent 50%, #b5302d 50%);
+  transition: background-position 0.3s cubic-bezier(0.19, 1, 0.22, 1) 0.1s,
+    color 0.5s ease 0s, background-color 0.5s ease;
 
-  &:hover{
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:24]: ok

File: src/sass/components/_button.scss [hunk 4]
```diff
@@ -16,29 +16,30 @@
...
-      color:rgba(255, 255, 255, 1);
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:25]: ok

File: src/sass/components/_button.scss [hunk 5]
```diff
@@ -16,29 +16,30 @@
 
   background-color: $accent-color;
   border: none;
-          
+
   background-size: 200% 100%;
-  background-image: linear-gradient(to right, transparent 50%, #B5302D 50%);
-  transition: background-position .3s cubic-bezier(0.19, 1, 0.22, 1) .1s, color .5s ease 0s, background-color .5s ease;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:22]: ok

File: src/sass/components/_button.scss [hunk 6]
```diff
@@ -16,29 +16,30 @@
 
   background-color: $accent-color;
   border: none;
-          
+
   background-size: 200% 100%;
-  background-image: linear-gradient(to right, transparent 50%, #B5302D 50%);
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:21]: ok

File: src/sass/components/_button.scss [hunk 7]
```diff
@@ -16,29 +16,30 @@
...
-  transition: background-position .3s cubic-bezier(0.19, 1, 0.22, 1) .1s, color .5s ease 0s, background-color .5s ease;
+  background-image: linear-gradient(to right, transparent 50%, #b5302d 50%);
+  transition: background-position 0.3s cubic-bezier(0.19, 1, 0.22, 1) 0.1s,
+    color 0.5s ease 0s, background-color 0.5s ease;
 
-  &:hover{
-      color:rgba(255, 255, 255, 1);
-      background-color:#B5302D;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:26]: ok

File: src/sass/components/_button.scss [hunk 8]
```diff
@@ -16,29 +16,30 @@
...
-      background-position: -100% 100%;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:27]: ok

File: src/sass/components/_button.scss [hunk 9]
```diff
@@ -16,29 +16,30 @@
...
+  &:hover {
+    color: rgba(255, 255, 255, 1);
+    background-color: #b5302d;
+    background-position: -100% 100%;
   }
 
   @media screen and (min-width: 768px) {
-      width: 294px;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:31]: ok

File: src/sass/components/_button.scss [hunk 10]
```diff
@@ -16,29 +16,30 @@
...
+    width: 294px;
   }
 
   @media screen and (min-width: 1440px) {
-      width: 400px;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:35]: ok

File: src/sass/components/_button.scss [hunk 11]
```diff
@@ -16,29 +16,30 @@
...
+    width: 400px;
   }
 }
 
-.btn__item-feedback{
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:39]: ok

File: src/sass/components/_button.scss [hunk 12]
```diff
@@ -16,29 +16,30 @@
...
-// use stiles "btn__item draw meet" to the item
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:40]: ok

File: src/sass/components/_button.scss [hunk 13]
```diff
@@ -16,29 +16,30 @@
...
-// <button class="btn__item-feedback draw meet">order</button>
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:41]: ok

File: src/sass/components/_button.scss [hunk 14]
```diff
@@ -52,99 +53,100 @@
 
   background: none;
   border: 0;
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:55]: ok

File: src/sass/components/_button.scss [hunk 15]
```diff
@@ -52,99 +53,100 @@
 
   background: none;
   border: 0;
-  
+
   box-shadow: inset 0 0 0 1px $accent-color;
   color: $primary-text-color;
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:58]: ok

File: src/sass/components/_button.scss [hunk 16]
```diff
@@ -52,99 +53,100 @@
...
   background: none;
   border: 0;
-  
+
   box-shadow: inset 0 0 0 1px $accent-color;
   color: $primary-text-color;
-  
+
   position: relative;
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:60]: ok

File: src/sass/components/_button.scss [hunk 17]
```diff
@@ -52,99 +53,100 @@
...
   box-shadow: inset 0 0 0 1px $accent-color;
   color: $primary-text-color;
-  
+
   position: relative;
-  
+
   &::before,
   &::after {
-      box-sizing: inherit;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:63]: ok

File: src/sass/components/_button.scss [hunk 18]
```diff
@@ -52,99 +53,100 @@
...
-      content: '';
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:64]: ok

File: src/sass/components/_button.scss [hunk 19]
```diff
@@ -52,99 +53,100 @@
...
-      position: absolute;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:65]: ok

File: src/sass/components/_button.scss [hunk 20]
```diff
@@ -52,99 +53,100 @@
...
-      width: 100%;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:66]: ok

File: src/sass/components/_button.scss [hunk 21]
```diff
@@ -52,99 +53,100 @@
...
-      height: 100%;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:67]: ok

File: src/sass/components/_button.scss [hunk 22]
```diff
@@ -52,99 +53,100 @@
...
+    box-sizing: inherit;
+    content: '';
+    position: absolute;
+    width: 100%;
+    height: 100%;
   }
 
   @media screen and (min-width: 768px) {
-      width: 294px;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:71]: ok

File: src/sass/components/_button.scss [hunk 23]
```diff
@@ -52,99 +53,100 @@
...
+    width: 294px;
   }
 
   @media screen and (min-width: 1440px) {
-      width: 400px;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:75]: ok

File: src/sass/components/_button.scss [hunk 24]
```diff
@@ -52,99 +53,100 @@
...
+    width: 400px;
   }
 }
-    
-  .draw {
-      transition: color 0.25s;
-  
+
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:79]: ok

File: src/sass/components/_button.scss [hunk 25]
```diff
@@ -52,99 +53,100 @@
...
+.btn__item-feedback--dimensions {
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:80]: ok

File: src/sass/components/_button.scss [hunk 26]
```diff
@@ -52,99 +53,100 @@
...
+  //  for btn SEND
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:81]: ok

File: src/sass/components/_button.scss [hunk 27]
```diff
@@ -52,99 +53,100 @@
...
+  @media screen and (min-width: 1440px) {
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:82]: ok

File: src/sass/components/_button.scss [hunk 28]
```diff
@@ -52,99 +53,100 @@
...
+    width: 510px;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:83]: ok

File: src/sass/components/_button.scss [hunk 29]
```diff
@@ -52,99 +53,100 @@
...
+    height: 68px;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:84]: ok

File: src/sass/components/_button.scss [hunk 30]
```diff
@@ -52,99 +53,100 @@
...
+  }
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:85]: ok

File: src/sass/components/_button.scss [hunk 31]
```diff
@@ -52,99 +53,100 @@
...
+}
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:86]: ok

File: src/sass/components/_button.scss [hunk 32]
```diff
@@ -52,99 +53,100 @@
...
+
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:87]: ok

File: src/sass/components/_button.scss [hunk 33]
```diff
@@ -52,99 +53,100 @@
...
+.draw {
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:88]: ok

File: src/sass/components/_button.scss [hunk 34]
```diff
@@ -52,99 +53,100 @@
...
+  transition: color 0.25s;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:89]: ok

File: src/sass/components/_button.scss [hunk 35]
```diff
@@ -52,99 +53,100 @@
...
+
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:90]: ok

File: src/sass/components/_button.scss [hunk 36]
```diff
@@ -52,99 +53,100 @@
...
   &::before,
   &::after {
-      border: 1px solid transparent;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:84]: ok

File: src/sass/components/_button.scss [hunk 37]
```diff
@@ -52,99 +53,100 @@
...
-      width: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:85]: ok

File: src/sass/components/_button.scss [hunk 38]
```diff
@@ -52,99 +53,100 @@
...
-      height: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:86]: ok

File: src/sass/components/_button.scss [hunk 39]
```diff
@@ -52,99 +53,100 @@
...
+    border: 1px solid transparent;
+    width: 0;
+    height: 0;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:88]: ok

File: src/sass/components/_button.scss [hunk 40]
```diff
@@ -52,99 +53,100 @@
...
-      width: 0;
-      height: 0;
+    border: 1px solid transparent;
+    width: 0;
+    height: 0;
   }
-  
+
   &::before {
-      top: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:90]: ok

File: src/sass/components/_button.scss [hunk 41]
```diff
@@ -52,99 +53,100 @@
...
-      left: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:91]: ok

File: src/sass/components/_button.scss [hunk 42]
```diff
@@ -52,99 +53,100 @@
...
+    top: 0;
+    left: 0;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:93]: ok

File: src/sass/components/_button.scss [hunk 43]
```diff
@@ -52,99 +53,100 @@
...
   &::before {
-      top: 0;
-      left: 0;
+    top: 0;
+    left: 0;
   }
-  
+
   &::after {
-      bottom: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:95]: ok

File: src/sass/components/_button.scss [hunk 44]
```diff
@@ -52,99 +53,100 @@
...
-      right: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:96]: ok

File: src/sass/components/_button.scss [hunk 45]
```diff
@@ -52,99 +53,100 @@
...
+    bottom: 0;
+    right: 0;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:98]: ok

File: src/sass/components/_button.scss [hunk 46]
```diff
@@ -52,99 +53,100 @@
...
   &::after {
-      bottom: 0;
-      right: 0;
+    bottom: 0;
+    right: 0;
   }
-  
+
   &:hover {
-      color: #b5302d;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:100]: ok

File: src/sass/components/_button.scss [hunk 47]
```diff
@@ -52,99 +53,100 @@
...
+    color: #b5302d;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:102]: ok

File: src/sass/components/_button.scss [hunk 48]
```diff
@@ -52,99 +53,100 @@
...
+
   &:hover {
-      color: #b5302d;
+    color: #b5302d;
   }
-  
+
   &:hover::before,
   &:hover::after {
-      width: 100%;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:105]: ok

File: src/sass/components/_button.scss [hunk 49]
```diff
@@ -52,99 +53,100 @@
...
-      height: 100%;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:106]: ok

File: src/sass/components/_button.scss [hunk 50]
```diff
@@ -52,99 +53,100 @@
...
+    width: 100%;
+    height: 100%;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:108]: ok

File: src/sass/components/_button.scss [hunk 51]
```diff
@@ -52,99 +53,100 @@
...
-      border-top-color: $primary-text-color;
-      border-right-color: $primary-text-color;
-      transition:
-      width 0.25s ease-out,
-      height 0.25s ease-out 0.25s;
+    border-top-color: $primary-text-color;
+    border-right-color: $primary-text-color;
+    transition: width 0.25s ease-out, height 0.25s ease-out 0.25s;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:116]: ok

File: src/sass/components/_button.scss [hunk 52]
```diff
@@ -52,99 +53,100 @@
...
+
   &:hover::before {
-      border-top-color: $primary-text-color;
-      border-right-color: $primary-text-color;
-      transition:
-      width 0.25s ease-out,
-      height 0.25s ease-out 0.25s;
+    border-top-color: $primary-text-color;
+    border-right-color: $primary-text-color;
+    transition: width 0.25s ease-out, height 0.25s ease-out 0.25s;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:121]: ok

File: src/sass/components/_button.scss [hunk 53]
```diff
@@ -52,99 +53,100 @@
...
-  
+
   &:hover::before {
-      border-top-color: $primary-text-color;
-      border-right-color: $primary-text-color;
-      transition:
-      width 0.25s ease-out,
-      height 0.25s ease-out 0.25s;
+    border-top-color: $primary-text-color;
+    border-right-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:120]: ok

File: src/sass/components/_button.scss [hunk 54]
```diff
@@ -52,99 +53,100 @@
...
   }
-  
+
   &:hover::before {
-      border-top-color: $primary-text-color;
-      border-right-color: $primary-text-color;
-      transition:
-      width 0.25s ease-out,
-      height 0.25s ease-out 0.25s;
+    border-top-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:119]: ok

File: src/sass/components/_button.scss [hunk 55]
```diff
@@ -52,99 +53,100 @@
...
   }
-  
+
   &:hover::after {
-      border-bottom-color: $primary-text-color;
-      border-left-color: $primary-text-color;
-      transition:
-      border-color 0s ease-out 0.5s,
-      width 0.25s ease-out 0.5s,
+    border-bottom-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:125]: ok

File: src/sass/components/_button.scss [hunk 56]
```diff
@@ -52,99 +53,100 @@
...
+    border-left-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:126]: ok

File: src/sass/components/_button.scss [hunk 57]
```diff
@@ -52,99 +53,100 @@
...
+    transition: border-color 0s ease-out 0.5s, width 0.25s ease-out 0.5s,
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:127]: ok

File: src/sass/components/_button.scss [hunk 58]
```diff
@@ -52,99 +53,100 @@
...
       height 0.25s ease-out 0.75s;
   }
 }
 
 .draw.meet {
-  
   &:hover {
-      color: #b5302d;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:130]: ok

File: src/sass/components/_button.scss [hunk 59]
```diff
@@ -52,99 +53,100 @@
...
+    color: #b5302d;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:132]: ok

File: src/sass/components/_button.scss [hunk 60]
```diff
@@ -52,99 +53,100 @@
...
 .draw.meet {
-  
   &:hover {
-      color: #b5302d;
+    color: #b5302d;
   }
-  
+
   &::after {
-      top: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:134]: ok

File: src/sass/components/_button.scss [hunk 61]
```diff
@@ -52,99 +53,100 @@
...
-      left: 0;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:135]: ok

File: src/sass/components/_button.scss [hunk 62]
```diff
@@ -52,99 +53,100 @@
...
+    top: 0;
+    left: 0;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:137]: ok

File: src/sass/components/_button.scss [hunk 63]
```diff
@@ -52,99 +53,100 @@
...
   &::after {
-      top: 0;
-      left: 0;
+    top: 0;
+    left: 0;
   }
-  
+
   &:hover::before {
-      border-top-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:139]: ok

File: src/sass/components/_button.scss [hunk 64]
```diff
@@ -52,99 +53,100 @@
...
-      border-right-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:140]: ok

File: src/sass/components/_button.scss [hunk 65]
```diff
@@ -52,99 +53,100 @@
...
+    border-top-color: $primary-text-color;
+    border-right-color: $primary-text-color;
   }
-
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:142]: ok

File: src/sass/components/_button.scss [hunk 66]
```diff
@@ -52,99 +53,100 @@
...
+
   &:hover::after {
-      border-bottom-color: $primary-text-color;
-      border-left-color: $primary-text-color;
-      transition:
-      height 0.25s ease-out,
-      width 0.25s ease-out 0.25s;
+    border-bottom-color: $primary-text-color;
+    border-left-color: $primary-text-color;
+    transition: height 0.25s ease-out, width 0.25s ease-out 0.25s;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:150]: ok

File: src/sass/components/_button.scss [hunk 67]
```diff
@@ -52,99 +53,100 @@
...
-  
+
   &:hover::after {
-      border-bottom-color: $primary-text-color;
-      border-left-color: $primary-text-color;
-      transition:
-      height 0.25s ease-out,
-      width 0.25s ease-out 0.25s;
+    border-bottom-color: $primary-text-color;
+    border-left-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:149]: ok

File: src/sass/components/_button.scss [hunk 68]
```diff
@@ -52,99 +53,100 @@
...
   }
-  
+
   &:hover::after {
-      border-bottom-color: $primary-text-color;
-      border-left-color: $primary-text-color;
-      transition:
-      height 0.25s ease-out,
-      width 0.25s ease-out 0.25s;
+    border-bottom-color: $primary-text-color;
```
Comment (oleksandrsoldatchenko) [inline src/sass/components/_button.scss:148]: ok

**Review by oleksandrsoldatchenko — COMMENTED** [review]
Пересмотрел правки. Окнул. Можно утверждать.

**Pull request merged**